libraries/helpers.rb

```diff
@@ -395,11 +389,22 @@ def default_site_template_source
       end
 
       # mod_php
+      def apache_mod_php_supported?
+        case node['platform_family']
+        when 'amazon', 'fedora'
+          false
+        when 'rhel'
+          !(node['platform_version'].to_i >= 9)
```

Comment:
Maybe this should be < 9 instead? Rather than inverting it.

Reply:
I feel like this might have been a cookstyle fix that I missed touching up after the fact. I will fix this!